docs/content/deploy.md

```diff
@@ -0,0 +1,121 @@
+**Warning:** The [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
+
+-----
+## Deploying your game
+
+When you want to try out your game, you need to run a deployment of the game. This means launching SpatialOS itself. SpatialOS sets up the game world and optionally starts up the server-workers needed to run the game world. Once the deployment is running, you can connect clients to it in order to play the game. You can run a deployment on your development machine (a "local deployment" or in the cloud (a "cloud deployment").
+
+Before you deploy your game, you need to build its workers; see documentation on [building workers](build.md).
+
```

Comment:
Add here:

This page contains:
* [How to deploy your game](how-to-deploy-your-game)
<br/> - Use this if you want to deploy your own game.
* [Quickstart: How to deploy the `Playground` project`](quickstart:-how-to-deploy-the-`Playground`-project`
<br/> - Use this if you want to experiment with deployments.

Reply:
We need to do this to explain why we have these two options on here. It's confusing otherwise!